### ISSUE TITLE
cylc review: NoneType job counts for latest cycle point

### ISSUE THREAD

**Issue by sadielbartholomew** [cylc/cylc-flow#3271]
**Describe the bug**

For suites run using Cylc 8.0a (regardless of the registration or start-up history) the most recent cycle point, i.e. the one at the top of 'cycles list' table, can display ``None`` rather than an integer for the per-status job count.

This Issue goes from the state of the codebase after #3269, which solves problem leading to similar downstream issues in Cylc Review, but this relates to the Cylc Review display rather than the ``job.sh`` extraction, hence a separate Issue to #3250 where this symptom could appear for any cycle point, not just the most recent.

Note it is not a case of premature display of the most recent cycle point, as I have observed there is always at least one element in that row which has data, usually the '# active tasks' element, even if all per-status '# jobs' columns all show ``None``.

**Release version(s) and/or repository branch(es) affected?**
Suites run using *only* Cylc 8.0a0 (the issue is not there for suites run on ``7.8.x``) using the ``7.8.3`` Cylc Review.

**Steps to reproduce the bug**
Run a suite which has quickly-completing cycles using Cylc 8.0a0, & quickly refresh to continuously observe the 'cycles list' page for it on Cylc Review. You should see ``None`` elements appear across the topmost cycle point row under any '# jobs' column, which get incrementally updated with the job count integer (if you refresh to catch them quick enough).

**Expected behavior**
An integer instead of ``None`` for the jobs columns & (following from this, but as a separate requirement) a consistent report format for suites run with different versions of Cylc.

**Screenshots**

Note here the encircled rows are to highlight the same cycle points across the different screenshots, which were taken consecutively for a single run.

![immediate_issue_marked](https://user-images.githubusercontent.com/30274190/62459870-59ad5100-b778-11e9-8735-2ef2a6cde40b.png)
![immediate_2_marked](https://user-images.githubusercontent.com/30274190/62459869-59ad5100-b778-11e9-9588-3ae30ffbd7b4.png)
![immediate_3_marked](https://user-images.githubusercontent.com/30274190/62459868-59ad5100-b778-11e9-8803-9c4b0c221004.png)

**Comment by wxtim** [comment]
No longer reproducible with 8.4.x
